index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+    <header>
+        <div>
+            <h3 class="logo">My Logo</h3>
+        </div>
+        <div class="menu">
+            <img src="./assets/icons/Union.png" alt="Menu Icon">
+        </div>
+    </header>
+
+    <main>
+        <section class="headline-section">
+            <h1 class="heading-1">I'm Abdul Glad to see you!</h1>
+            <p class="intro-info">I’m a software developer! I can help you build a product , feature or website Look
+                through some of my work and experience! If you like what you see and have a project you need coded,
+                don’t hestiate to contact me.</p>
+            <a class="connect-info">LET’S CONNECT</a>
```

Comment:
- [ ] Kindly make sure to wrap `Lets Connect` in hx tag like h2, h3, h4 etc.... Replace `a tag `with `hx tag`.